Util.java

```diff
@@ -0,0 +1,8 @@
+package org.usfirst.frc4904.standard;
+
+
+public class Util {
+	public static boolean checkZero(double var) {
```

Comment:
Or perhaps isZeroFPASafe or something else to indicate when/why it should be used. Definitely Javadoc it.